src/app/MessageDef.h

```diff
@@ -1865,6 +1939,136 @@ class Builder : public chip::app::Builder
 };
 }; // namespace InvokeCommand
 
+namespace ReadRequest {
```

Comment:
same here: for maintainability, please split up code into independent chunks. This way things can get included/built/unittested independently.

Reply:
https://github.com/project-chip/connectedhomeip/issues/3644

Reply:
Done, thanks